event_email_reminder/models/event.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# © 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, fields, models
+from datetime import datetime, timedelta
+
+from openerp.tools import DEFAULT_SERVER_DATE_FORMAT
+
+
+class EventEvent(models.Model):
+    _inherit = 'event.event'
+
+    @api.model
+    def run_event_email_reminder(
+            self, days=7, near_events=False, template_id=None):
+        """
+        @param days: number of days to reminder when events star
+        @param near_events: If you want receive the events which start
+        between now and limit date
+        @param template_name: Name of a template or None
+        @return:
```

Comment:
I think the standard is to use [Sphinx syntax](http://www.sphinx-doc.org/en/stable/domains.html) for this kind of definitions:

```
:param int days:
    Number of days to reminder when events star
:param bool near_events:
    If you want receive the events which start
etc.
```